source/common/filter/ratelimit.cc

```diff
@@ -43,7 +43,7 @@ Network::FilterStatus Instance::onNewConnection() {
     config_->stats().active_.inc();
     config_->stats().total_.inc();
     calling_limit_ = true;
-    client_->limit(*this, config_->domain(), config_->descriptors(), EMPTY_STRING);
+    client_->limit(*this, config_->domain(), config_->descriptors(), {"", ""});
```

Comment:
yeah in this case for some reason that does not work, compiler complains about missing initializer for request_id and span_context_